tests/unit/test-firefox/test.firefox.js

```diff
@@ -38,14 +38,11 @@ describe('firefox', () => {
     };
 
     function createFakeFxRunner(firefoxOverrides = {}) {
-      const firefox = {
+      const process = {
```

Comment:
How about calling it `firefoxProcess` or `fxProcess` instead? 
`process` is going to shadow the `process` identifier as defined by nodejs 
(that's a pity that the eslint `no-shadow` rule doesn't seem to have any config available for customized disallowed identifier)